index.html

```diff
@@ -0,0 +1,841 @@
+            <li><a href="#">Terms & Conditions</a></li>
+          </ul>
+
+          <ul class="navlist-top2 dflex">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
+          </ul>
+        </div>
+        <div class="dflex-justify-space-center navbar-xl-bottom font-1">
+          <div>
+            <ul class="navlist2 navlist-list1 dflex">
+              <li><a href="#">Latest</a></li>
+              <li><a href="#">Hard Fork</a></li>
+              <li><a href="#">Plugged</a></li>
+              <li><a href="#">ReadMe</a></li>
+              <li><a href="#">Growth Quarters</a></li>
+            </ul>
+          </div>
+          <div>
+            <i class='fa fa-search'></i>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <nav class="xxl">
+      <div class="dflex-justify-space-center navbar-xl-top">
+        <ul class="navlist-list1 dflex">
+          <li><a href="#">News</a></li>
+          <li>
+            <a href="#">Events <i class="fas fa-caret-down"></i></a>
+          </li>
+          <li>
+            <a href="#">Business <i class="fas fa-caret-down"></i></a>
+          </li>
+          <li><a href="#">AMAs</a></li>
+          <li><a href="#">Spaces</a></li>
+          <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+
+        <ul class="navlist-top2 dflex">
+          <li><a href="#">About</a></li>
+          <li><a href="#">Advertise</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class='second-nav-mobile mobile'>
+      <p>
+        Join TNW2020, our 15th annual flagship event. Online . Get tickets ➔
+      </p>
+    </nav>
+  </header>
+  <main>
+    <div class="fixed-nav">
+      <div class="dflex-justify-space-center navbar-xl-bottom font-1">
+        <div>
+          <ul class="navlist2 navlist-list1 dflex">
+            <li><a href="#">LATEST</a></li>
+            <li><a href="#">HARD FORK</a></li>
+            <li><a href="#">PLUGGED</a></li>
+            <li><a href="#">README</a></li>
+            <li><a href="#">GROWTH QUARTERS</a></li>
+            <li><a href="#">SHIFT</a></li>
+            <li><a href="#">NEURAL</a></li>
+          </ul>
+        </div>
+        <div class="nav-buttons-xxl dflex-justify-center">
+          <i class='fa fa-search'></i>
+          <img src="./images/event-bl-tnw2020.svg" alt="2020">
+          <p class="common-color date-time"> 22 :07 :33 :25 </p>
+          <button type="button">GET TICKETS</button>
+        </div>
+      </div>
+      <nav class='second-nav-mobile xxl'>
+        <p>
+          Join TNW2020, our 15th annual flagship event. Online . Get tickets ➔
+        </p>
+      </nav>
+    </div>
+    <section class="first-article">
+      <div class='first-article-cards-small article-card-ipad-flex'>
```

Comment:
Let's reduce the space here 👇 .
![image](https://user-images.githubusercontent.com/57726348/92767614-ea5f5300-f3b4-11ea-84da-65b46a17b227.png)